docs/API.md

```diff
@@ -1551,6 +1555,82 @@ Example: `GET ~bob/trusty/wordpress-42/meta/id-series`
 }
 ```
 
+#### GET *id*/meta/home-page
+
+The `meta/home-page` path returns the home page project of 
+the entity if any specified.
+
+```go
+type HomePageResponse struct {
+        HomePage string
+}
+```
+
+Example: `GET bundle/mediawiki/meta/home-page`
+
+```json
+{
+    "HomePage": "http://mediawiki.org"
+}
+```
+
+#### PUT *id*/meta/home-page
+
+This request updates the home page associated with the charm or bundle.
+
+```go
+type HomePage struct {
+        HomePage string
+}
+```
+
+Example: `PUT precise/wordpress-32/meta/home-page`
+
+Request body:
+```json
+{
+    "HomePage": "http://newmediawiki.org"
+}
+```
```

Comment:
Could you please specify what's returned in the response, even if nothing is returned?